CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 ### FEATURES
 
 - [\#346](https://github.com/cosmos/evm/pull/346) Add eth_createAccessList method and implementation
+- [\#502](https://github.com/cosmos/evm/pull/502) Add block time in derived logs.
```

Comment:
Could you permalink to the `proto` file that this is updating?  Is this change API / runtime breaking at all?

Reply:
https://github.com/cosmos/evm/blob/3e3793173dff802dac1723443e54affb4304f714/proto/cosmos/evm/vm/v1/evm.proto#L245

the proto change is only for internal APIs between json-rpc and grpc.